versionwarning.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+    // adapted 2020-05 from https://scikit-learn.org/versionwarning.js
+    var latestStable = '0.20.5';
```

Comment:
On our website we don't differentiate between point releases. Let's just go with `0.20` so that we don't have to update with every point release (right now it's very easy to do point releases and I'd rather not add an extra step of updating this)